apps/protocol-api/src/prisma/resolvers/Guild.ts

```diff
@@ -57,15 +131,15 @@ export class GuildCustomResolver {
         FROM (
           SELECT dt::date
           FROM generate_series(${lookbackDate}, ${startDate}, '1 day'::interval) dt
-        ) d 
+        ) d
         LEFT JOIN "Contribution" as c
           ON c.date_of_engagement::date = d.dt::date
-        LEFT JOIN "GuildContribution" as gc 
+        LEFT JOIN "GuildContribution" as gc
           ON gc."contribution_id" = c.id
         WHERE (
           gc."guild_id" IS NULL OR gc."guild_id" = ${guildId}
```

Comment:
**Side note**: 
this field is declared as non-null in prisma's schema. Is this check necessary?

Reply:
Yes, because a contribution might not have a guild_id